config/locales/en.yml

```diff
@@ -180,8 +180,8 @@ en:
         title: "Follow-up patients"
         subtitle: "Patients that had a BP taken, a blood sugar taken, an appointment scheduled, or their medications updated at %{facility_name}."
       patients_initiated_on_treatment_card:
-        title: 'Where is the "Patients initiated on treatment" data?'
-        main_text: 'Use "Registered patients", it will always equal "Patients initiated on treatment" for patients registered on Simple.'
+        title: 'Where are the "Patients initiated on treatment" data?'
```

Comment:
This should be "is", I think - since we're asking "Where is the data?"

Reply:
My bad :(